test/core/end2end/tests/connectivity.cc

```diff
@@ -33,6 +33,12 @@ typedef struct {
   grpc_completion_queue* cq;
 } child_events;
 
+typedef struct {
```

Comment:
Suggest giving this struct a simple constructor, so the values can be initialized here instead of in the test function.